qa-tests-backend/src/test/groovy/ExternalNetworkSourcesTest.groovy

```diff
@@ -95,7 +105,7 @@ class ExternalNetworkSourcesTest extends BaseSpecification {
 
         log.info "Create a smaller network external source containing Cloudflare's IP address"
         String externalSource31Name = generateNameWithPrefix("external-source-31")
-        NetworkEntity externalSource31 = createNetworkEntityExternalSource(externalSource31Name, CF_CIDR_31)
+        NetworkEntity externalSource31 = createNetworkEntityExternalSource(externalSource31Name, smallCIDR)
```

Comment:
All those names with 31 and 30 would then loose their meaning as they were referring to `/30` and `/31` masks. Do you think we should update them to something generic like, for example, `supernet` and `subnet`?